Ex01_Files01/src/lt/verbus/Person.java

```diff
@@ -0,0 +1,39 @@
+package lt.verbus;
+
+public class Person {
+    private int id;
+    private String name;
+    private String surname;
+    private double receivedMoney;
+    private double sentMoney;
+
+    public Person(int id, String name, String surname) {
+        this.id = id;
+        this.name = name;
+        this.surname = surname;
+        this.receivedMoney = 0;
+        this.sentMoney = 0;
+    }
+
+    public void transactionIn(double amount) {
```

Comment:
set...., set... get..., is..., of... priduoda programuotojui aiskumo. pvz setTransactionIn(double amount)